### PR TITLE
Add my keymap for massdrop/ctrl

### PR DESCRIPTION
**Pull request by alexschneider**
## Description

This is a rather small change from the endgame keymap based on my own personal preferences, so I'm unsure if a PR is appropriate. I'm happy to close if it is not. 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/massdrop/ctrl/keymaps/alexschneider/config.h [hunk 1]
```diff
@@ -0,0 +1,116 @@
+#pragma once
```
Comment (drashna) [inline keyboards/massdrop/ctrl/keymaps/alexschneider/config.h:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/massdrop/ctrl/keymaps/alexschneider/config_led.c [hunk 2]
```diff
@@ -0,0 +1,82 @@
+#ifdef RGB_MATRIX_ENABLE
```
Comment (drashna) [inline keyboards/massdrop/ctrl/keymaps/alexschneider/config_led.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/massdrop/ctrl/keymaps/alexschneider/keymap.c [hunk 3]
```diff
@@ -0,0 +1,382 @@
+#include "keymap.h"
```
Comment (drashna) [inline keyboards/massdrop/ctrl/keymaps/alexschneider/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/massdrop/ctrl/keymaps/alexschneider/keymap.h [hunk 4]
```diff
@@ -0,0 +1,98 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/massdrop/ctrl/keymaps/alexschneider/keymap.h:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/massdrop/ctrl/keymaps/alexschneider/config_led.c (hunk 2 above)
Comment (fauxpark) [inline keyboards/massdrop/ctrl/keymaps/alexschneider/config_led.c:1]: I don't think you need this file at all. It's identical to the one in the keyboard folder.

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Pull request closed without merging**